EIPS/eip-99999.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 99999
+title: Builder generalized consolidation requests.
+description: Adds a new generalized consolidation requests to create builders.
+author: Potuz (@potuz)
+discussions-to: https://ethereum-magicians.org/t/eip-99999-generalized-consolidation-requests/11111
```

Comment:
Please create a discussions topic in Eth Magicians using the template: https://ethereum-magicians.org/c/eips/5